mappings/net/minecraft/client/render/MapRenderer.mapping

```diff
@@ -0,0 +1,36 @@
+CLASS net/minecraft/class_330 net/minecraft/client/render/MapRenderer
+	FIELD field_2043 textureManager Lnet/minecraft/class_1060;
+	FIELD field_2044 MAP_ICONS_TEXTURE Lnet/minecraft/class_2960;
+	FIELD field_2045 mapTextures Lit/unimi/dsi/fastutil/ints/Int2ObjectMap;
+	FIELD field_21688 MAP_ICONS_RENDER_LAYER Lnet/minecraft/class_1921;
+	METHOD <init> (Lnet/minecraft/class_1060;)V
+		ARG 1 textureManager
+	METHOD method_1769 updateTexture (ILnet/minecraft/class_22;)V
+		ARG 1 id
+		ARG 2 state
+	METHOD method_1771 clearStateTextures ()V
+	METHOD method_1773 draw (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;ILnet/minecraft/class_22;ZI)V
+		ARG 1 matrices
+		ARG 2 vertexConsumers
+		ARG 3 id
+		ARG 4 state
+		ARG 5 hideNonAlwaysRenderedIcons
```

Comment:
```suggestion
		ARG 5 hideOptionalIcons
```
is this a correct guess?

Reply:
The list of hidden icons are:
 - `PLAYER`
 - `RED_MARKER` (unused)
 - `BLUE_MARKER` (unused)
 - `PLAYER_OFF_MAP`
 - `PLAYER_OFF_LIMITS`
 
 The boolean is usually true if it's on an item frame or in a cartography table, so I don't know if "optional" is fitting.
 
 I was thinking of `hidePlayerIcons` but I'm scared it could be used for other stuff in the future
 
 `RED_MARKER` and `BLUE_MARKER` are like re-colored `PLAYER` textures.

Reply:
Hmm, if this boolean is true if and only if for a non-player context (a map not on a player), you can consider naming from that perspective